JHUGenLexicon/src/JHUGenLexiconTranslator.cc

```diff
@@ -42,94 +45,133 @@ void JHUGenLexiconTranslator::translate(){
     }
   }
   // Fix output by subracting by constant vectors
-
+  
   // Offsets for input Amplitude JHUGen 
   if (basis_input == bAmplitude_JHUGen && basis_output == bHiggsBasis){
-    voutput.at(coupl_hbasis_dCz).first -= 1.0;
-    voutput.at(coupl_hbasis_dCw).first -= 1.0;
+     voutput.at(coupl_hbasis_dCz).first -= 1.0;
+     voutput.at(coupl_hbasis_dCw).first -= 1.0;
   }
   // Offsets for EFT_JHUGen inputs
-  else if (basis_input == bEFT_JHUGen && basis_output == bAmplitude_JHUGen && include_triple_quartic_gauge){
```

Comment:
Is it possible to keep the newline and spacing formatting?
New lines should be LF
Spacing should be 2 spaces, no tabs. Please only change the code, not the formatting.

Reply:
Also, please keep the file permissions as they are and avoid overwriting via manual upload.

Reply:
I cannot keep the new line and spacing format, since I wasn't working in the lexicon branch for the original bug fixes I made, so I uploaded to this branch with the file overwrite. What exactly is the spacing format that you had earlier? Is it the one in master? I can try and mimic that and re submit the pull.

Reply:
I believe @hroskes had a way to do this automatically. Maybe he can comment. You can also write a python script to pass over a diff of the files, trim preceding and trailing whitespace in the two versions and update.

Yes, formatting in the master branch would be preferred.

Reply:
Ok, in the meantime, to save time waiting I just did :set et|retab in vim to remove all tabs and then :%s/\s\+$//e to trim whitespace at the end of lines. I have to go through and fix the 2 spaces manually but I should be done within 30 min.

Reply:
I fixed the formatting. Check to see if it is satisfactory